src/Footer.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+
+const Footer = props => {
+  return (
+    <footer>
+      <ul className="footer-contact">
+        {props.contactDetails.map((element, index) => (
+          <li key={index}>{element}</li>
```

Comment:
It's best to try and avoid using the array `index` as a `key`. I know this is mentioned in the syllabus but have a read of this if you are not aware already https://legacy.reactjs.org/docs/lists-and-keys.html#keys

The react docs can do a much better explanation then me 😊